### ISSUE TITLE
Installer not working on Windows 8.1

### ISSUE THREAD

**Issue by MrXeious** [atom/atom#7839]
I tried to install Atom 1.0 after downloading it but when I click at the icon of the installer a window appear that shows Atom is being installed and when it disappeared there's nothing happened.
> I'm using windows 8.1 by the way.

**Comment by Zireael07** [comment]
Change the issue name to something like "Installer not working on Windows 8.1". Otherwise it's not informative at all.

**Comment by mnquintana** [comment]
Can you provide a bit [more information](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#submitting-issues)?

**Comment by batjko** [comment]
How long does it take after you see the installer, until it disappears? Is it within 5 seconds or so, or may it have had enough time to install the whole thing?

**Comment by queq** [comment]
I'm on Windows 8.1 too, and this window appears after executing the installer:
![exception](https://cloud.githubusercontent.com/assets/4623040/8658227/22f50cfe-2968-11e5-968d-e9953db9315a.png)

**Comment by jgopel** [comment]
I'm on windows 8.1 as well and experiencing a similar issue.

I have been running atom since before 1.0.0. The auto-update to 1.0.0 went fine for me. When I went to update to 1.0.2 (missed 1.0.1), 1.0.0 was uninstalled, but nothing was reinstalled in its place. There was no error message.

I downloaded the 1.0.2 installer from atom.io, which fails almost immediately on launch with no error message. I downloaded the 1.0.1 installer, which ran fine.

To my surprise, however, when I checked the version number inside atom, version 1.0.2 is listed. I'm not sure how this could be, as it seems that the install has never succeeded. I have not noticed any odd behavior, but it may be that the versions are so close to each other that nothing would have manifested yet.

Let me know if this should be its own issue.
